CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -16,6 +16,8 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 ### Dependencies
 - Compile binaries using [go 1.21.6](https://github.com/etcd-io/etcd/pull/17368).
 - Upgrade [bbolt to 1.3.9](https://github.com/etcd-io/etcd/pull/17484).
+- Compile binaries using [go 1.21.8](https://github.com/etcd-io/etcd/pull/17538).
+- Update go version to 1.21.8 for build. [go 1.21.8](https://github.com/etcd-io/etcd/pull/17541).
```

Comment:
I think we can drop the second line.  In my view we would only need the release note for the compiled binaries.

Reply:
Thanks, done.